README.md

```diff
@@ -1,55 +1,80 @@
 
-A unified programming model for event-sourced processing against stream-based stores including snapshots, caching and other state management/retrieval optimizations. Does not and will not handle projections, subscriptions etc. (see [Propulsion](https://github.com/jet/propulsion) for that).
-
-Focused on remaining a set of low dependency _libraries_ that _you_ compose into an architecture that fits your apps needs; not a final Best Practice Architecture/object model/processing pipeline foisting a one-size-fits-all model on you. You decide what facilities make sense for your context; Equinox covers just the selected infrastructural aspects. (That's not to say we don't have plenty opinions and well polished patterns; we just try to confine the impact of that to [`samples`](/samples) or [templates](https://github.com/jet/dotnet-templates), leaving judgement calls open for you to adjust as your app evolves). This core repo will never feature tight integration with any specific web or services framework.
-
-The design is informed by discussions, talks and countless hours of hard and thoughtful work invested into many previous systems, [frameworks](https://github.com/NEventStore), [samples](https://github.com/thinkbeforecoding/FsUno.Prod), [forks of samples](https://github.com/bartelink/FunDomain), the outstanding continuous work of the [EventStore](https://github.com/eventstore) founders and team and the wider [DDD-CQRS-ES](https://groups.google.com/forum/#!forum/dddcqrs) community. It would be unfair to single out even a small number of people despite the immense credit that is due.
+A unified programming model for event-sourced processing against stream-based stores including snapshots, caching and other state management/retrieval optimizations. The library does not and will not handle projections, subscriptions etc. (see [Propulsion](https://github.com/jet/propulsion) for that).
+
+Equinox is focused on remaining a set of low dependency _libraries_ that _you_ compose into an architecture that fits your apps needs. This means that Equinox can also help with general accesss strategies for your storage options. The library is neither a source for Best Practice Architecture/object model/processing pipeline nor is it imposing a one-size-fits-all model on you. You decide what facilities make sense for your context and Equinox covers the infrastructural aspects. If you are new to event-sorucing we do have sample patterns that are located in [`samples`](/samples) or [templates](https://github.com/jet/dotnet-templates). This core repo will never feature tight integration with any specific web or services framework.
```

Comment:
```suggestion
Equinox is focused on remaining a set of low dependency _libraries_ that _you_ compose into an architecture that fits your apps needs. The library is neither a source for Best Practice Architecture/object model/processing pipeline nor is it imposing a one-size-fits-all model on you. You decide what facilities make sense for your context and Equinox covers the infrastructural aspects. If you are new to event-sourcing we do have sample patterns that are located in [`samples`](/samples) or [templates](https://github.com/jet/dotnet-templates). This core repo will never feature tight integration with any specific web or services framework.
```

>This means that Equinox can also help with general access strategies for your storage options.

I dont know what this means

Reply:
This intro paragraph really needs to explain the things that we are pushing for within the team. It's clear that equinox is used not only as an event sourcing library but also to do general transactions with the data store. That needs to be explicit if we're going to continue that